recursive_remove.cc: fail to build

I'm trying to build for arm64 (in docker, ubuntu jammy), but getting the following error:

> /tmp/saunafs_deb_working_directory.2024-05-07_143835.1Z3oVs/saunafs/src/tools/recursive_remove.cc: In function 'int recursive_remove_run(int, char**)':
> /tmp/saunafs_deb_working_directory.2024-05-07_143835.1Z3oVs/saunafs/src/tools/recursive_remove.cc:129:47: error: comparison is always true due to limited range of data type [-Werror=type-limits]
>   129 |         while ((ch = getopt(argc, argv, "l")) != -1) {
>       |                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~
> cc1plus: all warnings being treated as errors

It seems that the issue is not related to the architecture, but rather that `ch` is unsigned.

--- a/src/tools/recursive_remove.cc
+++ b/src/tools/recursive_remove.cc
@@ -122,7 +122,7 @@ static int recursive_remove(const char *file_name, int long_wait) {
 }
 
 int recursive_remove_run(int argc, char **argv) {
-	char ch;
+	int ch;
 	int status;
 	int long_wait = 0;
 

